DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -45,11 +45,17 @@
     ".hepmc3", ".hepmc3.gz", ".hepmc3.xz", ".hepmc3.bz2",
     ".hepmc3.tree.root",
     ]
+EDM4HEP_INPUT_EXTENSIONS = [
+    ".root",
```

Comment:
Wondering whether we should add a comment here, that these have to be checked **after** the HepMC3 extensions to make sure that we do not actually try to open a HepMC3 root file with the EDM4hep reader, if someone has the idea to refactor the `DD4hepSimulation` below. Or at least we should make sure that the tests catch that (which might be the case already).

Reply:
Not here, but when we try to figure out the inputAction, I think?

Reply:
Comment coming in the next commit.